minigalaxy/launcher.py

```diff
@@ -88,6 +91,18 @@ def get_windows_exe_cmd(game, files):
         executables.remove(os.path.join(game.install_dir, "unins000.exe"))
         filename = os.path.splitext(os.path.basename(executables[0]))[0] + '.exe'
         exe_cmd = ["wine", filename]
+
+    # Check if variable environment are set.
+    # Command can be empty
+    if var[0] != "":
+        while i < len(var):
+            if i < len(var):
+                exe_cmd.insert(i, var[i])
+                i = i + 1
+            else:
+                break
+
+    exe_cmd.extend(command)
```

Comment:
This can be done easier. Be aware that index 0 might not be present in var (var_list), so this first "if" is risky.
exe_cmd = var_list + exe_cmd + command_list

Reply:
Change done, TY